dcat3-implementation-report/table-implementations.html

```diff
@@ -1,89 +1,26 @@
+    <tr id="imp-REV6" class="">	<td>	<a href="#REV6">REV6</a>	</td><td>		</td><td>		</td><td>		</td><td><code>	dcat:Resource	</code></td><td><a data-cite="VOCAB-DCAT-3#Property:resource_has_part"><code>	dcterms:hasPart	</code></a></td><td>		</td><td class="nr success">	2	</td><td class="nr warn">	1	</td><td>	&#x2714;</td><td>(&#x2714;)</td><td></td><td>&#x2714;	</td>	</tr>
+    <tr id="imp-REV7" class="">	<td>	<a href="#REV7">REV7</a>	</td><td>		</td><td>		</td><td>		</td><td><code>	dcat:Resource	</code></td><td><a data-cite="VOCAB-DCAT-3#Property:resource_previous_version"><code>	dcat:previousVersion	</code></a></td><td>		</td><td class="nr success">	1	</td><td class="nr warn">	0	</td><td>	</td><td></td><td>&#x2714;</td><td>	</td>	</tr>
+    <tr id="imp-REV8" class="">	<td>	<a href="#REV8">REV8</a>	</td><td>		</td><td>		</td><td>		</td><td><code>	dcat:Resource	</code></td><td><a data-cite="VOCAB-DCAT-3#Property:resource_has_version"><code>	dcat:hasVersion	</code></a></td><td>		</td><td class="nr success">	4	</td><td class="nr warn">	0	</td><td>	&#x2714;</td><td>&#x2714;</td><td>&#x2714;</td><td>&#x2714;	</td>	</tr>
+    <tr id="imp-REV9" class="">	<td>	<a href="#REV9">REV9</a>	</td><td>		</td><td>		</td><td>		</td><td><code>	dcat:Resource	</code></td><td><a data-cite="VOCAB-DCAT-3#Property:resource_replaces"><code>	dcterms:replaces	</code></a></td><td>		</td><td class="nr success">	2	</td><td class="nr warn">	0	</td><td>	</td><td></td><td>&#x2714;</td><td>&#x2714;	</td>	</tr>
+    <tr id="imp-REV10" class="">	<td>	<a href="#REV10">REV10</a>	</td><td>		</td><td>		</td><td>		</td><td><code>	dcat:Resource	</code></td><td><a data-cite="VOCAB-DCAT-3#Property:resource_has_current_version"><code>	dcat:hasCurrentVersion	</code></a></td><td>		</td><td class="nr success">	1	</td><td class="nr warn">	0	</td><td>	</td><td></td><td>&#x2714;</td><td>	</td>	</tr>
+    <tr id="imp-REV11" class="">	<td>	<a href="#REV11">REV11</a>	</td><td>		</td><td>		</td><td>		</td><td><code>	dcat:Resource	</code></td><td><a data-cite="VOCAB-DCAT-3#Property:resource_version"><code>	dcat:version	</code></a></td><td>		</td><td class="nr success">	4	</td><td class="nr warn">	0	</td><td>	&#x2714;</td><td>(&#x2714;)</td><td>&#x2714;</td><td>&#x2714;	</td>	</tr>
+    <tr id="imp-REV12" class="">	<td>	<a href="#REV12">REV12</a>	</td><td>		</td><td>		</td><td>		</td><td><code>	dcat:Resource	</code></td><td><a data-cite="VOCAB-DCAT-3#Property:resource_version"><code>	adms:versiontes	</code></a></td><td>		</td><td class="nr success">	4	</td><td class="nr warn">	0	</td><td>	&#x2714;</td><td>&#x2714;</td><td>&#x2714;</td><td>&#x2714;	</td>	</tr>
```

Comment:
|ERRATA | CORRIGE |
|--|--|
`adms:versiontes` | `adms:versionNotes`

Reply:
Thanks Andrea, I have fixed this.